modules/apim-adapter/src/main/java/com/axway/apim/api/model/OutboundProfile.java

```diff
@@ -15,41 +15,44 @@
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
 
 public class OutboundProfile extends Profile {
-	
+
 	protected static Logger LOG = LoggerFactory.getLogger(OutboundProfile.class);
-	
+
 	String routeType;
-	
-	@JsonDeserialize( using = PolicyDeserializer.class)
+
+	@JsonDeserialize(using = PolicyDeserializer.class)
 	Policy requestPolicy;
-	
-	@JsonDeserialize( using = PolicyDeserializer.class)
+
+	@JsonDeserialize(using = PolicyDeserializer.class)
 	Policy responsePolicy;
-	
-	@JsonDeserialize( using = PolicyDeserializer.class)
+
+	@JsonDeserialize(using = PolicyDeserializer.class)
 	Policy routePolicy;
-	
-	@JsonDeserialize( using = PolicyDeserializer.class)
+
+	@JsonDeserialize(using = PolicyDeserializer.class)
 	Policy faultHandlerPolicy;
-	
+
 	String authenticationProfile;
-	
+
 	List<Object> parameters = new ArrayList<Object>();
 
 	public OutboundProfile() throws AppException {
 		super();
 	}
 
 	public String getAuthenticationProfile() {
-		return this.authenticationProfile;
+		if (StringUtils.isBlank(authenticationProfile))
+			return "_default";
+		else
+			return this.authenticationProfile;
 	}
 
 	public void setAuthenticationProfile(String authenticationProfile) {
 		this.authenticationProfile = authenticationProfile;
 	}
 
 	public String getRouteType() {
-		if(this.routePolicy!=null && !this.routePolicy.equals("")) {
+		if ((StringUtils.isNotBlank(routeType) && !StringUtils.equals("proxy", routeType)) || (routePolicy != null)) {
 			return "policy";
 		} else {
 			return "proxy";
```

Comment:
I know I'm also not adding comments in all cases, but can you perhaps add a comment to explain this.

Reply:
done